pkg/authorization/registry/rolebindingrestriction/etcd.go

```diff
@@ -34,7 +33,7 @@ func makeStore(optsGetter restoptions.Getter) (*registry.Store, error) {
 		ObjectNameFunc: func(obj runtime.Object) (string, error) {
 			return obj.(*authorizationapi.RoleBindingRestriction).Name, nil
 		},
-		PredicateFunc: func(label labels.Selector, field fields.Selector) *generic.SelectionPredicate {
+		PredicateFunc: func(label labels.Selector, field fields.Selector) storage.SelectionPredicate {
```

Comment:
Squash this to https://github.com/openshift/origin/pull/12143/commits/95baa45e497dc5216731280d7ee0c65bc20e37e6, it's the same.